src/lib.rs

```diff
@@ -1082,6 +1082,75 @@ fn digraph_floyd_warshall_numpy(
     Ok(mat.into_pyarray(py).into())
 }
 
+/// Collect runs that match a filter function
+///
+/// A run is a path of nodes where the is onlyt a single successor an all
```

Comment:
typos

Reply:
Oops, yep. Fixed those in: https://github.com/Qiskit/retworkx/pull/206/commits/58234a9e35ef86ed6bf999e2dad33bf09768a005